app/views/confirmables/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "confirmables/shared/links" %>
```

Comment:
Hi @Trust-007 

Nice job so far implementing the different project requirements but just some little adjustments to completely meet up the project requirements.

- [ ] Please modified your devise `login` and `registration` view file to match the wireframes and their styling.
![image](https://user-images.githubusercontent.com/55596970/224085289-bf5216cb-de8f-407f-8ed0-f080380934fc.png)

Reply:
@CollinsTatang Thanks for the suggestion.